packages/netlify-plugin-search-index/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@netlify/plugin-search-index",
+  "private": true,
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\""
+  },
+  "keywords": [
+    "netlify",
+    "netlify-plugin"
+  ],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@netlify/zip-it-and-ship-it": "^0.3.1",
```

```suggestion
   "@netlify/zip-it-and-ship-it": "^0.4.0-5",
```

That release includes `https://github.com/netlify/zip-it-and-ship-it/pull/48` which solves many bugs.